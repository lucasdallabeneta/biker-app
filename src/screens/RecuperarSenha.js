/* eslint-disable prettier/prettier */

import React, { Component } from 'react';
import {
  View,
  Text,
} from 'react-native';

export default class Main extends Component {
  static navigationOptions = {
    title: 'Recuperar Senha',
  };

  render() {
    return (
      <View style={{ flex:1 }}>
        <Text>
          TELA PRA RECUPERAR A SENHA
        </Text>
      </View>
    );
  }
}
