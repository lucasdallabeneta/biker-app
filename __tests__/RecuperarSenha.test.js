import React from 'react';
import RecuperarSenha from '../src/screens/RecuperarSenha';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<RecuperarSenha />).toJSON();
  expect(tree).toMatchSnapshot();
});