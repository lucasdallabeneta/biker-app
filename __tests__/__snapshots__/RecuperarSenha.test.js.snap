// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "flexDirection": "column",
      }
    }
  >
    <Text>
      Digite seu CPF
    </Text>
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      autoCorrect={false}
      onChangeText={[Function]}
      placeholder="CPF"
      placeholderTextColor="#aaa"
      rejectResponderTermination={true}
      underlineColorAndroid="transparent"
    />
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text>
        Enviar
      </Text>
    </View>
  </View>
</View>
`;
